setup.py

```diff
@@ -53,7 +54,7 @@ def run(self):
 
 setup(
     name='sphinx_rtd_theme',
-    version='0.4.3.dev0',
+    version=open(path.join('sphinx_rtd_theme', 'VERSION')).read().strip(),
```

Comment:
With setuptools ≥ 39.2.0, you can just add this to `setup.cfg`:
```ini
[metadata]
version = file: sphinx_rtd_theme/VERSION
```

And it will take care of reading and stripping for you. See pypa/setuptools#1359.

Reply:
Ah thanks, much easier.